galaxy_importer/collection.py

```diff
@@ -135,9 +130,27 @@ def _build_docs_blob(self):
             for c in self.content_objs
         ]
 
+        readme = markup_utils.get_readme_doc_file(self.path)
+        if not readme:
+            msg = 'No collection readme found'
+            self.log.error(msg)
+            raise exc.ImporterError(msg)
```

Comment:
Which part is responsible for handing this error? Maybe message can be logged there, otherwise it can easily lead to logging message in both places.

Reply:
I will remove the log message here, that will also match how we raise other `ImporterError` exceptions.

When these `ImporterError`s are raised, they are handled by the caller, outside of `galaxy-importer`, via change made in #17